### ISSUE TITLE
TTL?

### ISSUE THREAD

**Issue by bclaymiles** [mk-fg/tinydns-dynamic-dns-updater#1]
Hey there, just tried this out, it's a neat idea - but if I set a TTL on the records, the server app dies with 'ValueError: too many values to unpack'. 

Without a TTL, one is forced to wait on djb's default TTL's, which are much too long to respond to a dynamic change. 

Is there a workaround -- a way to force a lower TTL?

**Comment by mk-fg** [comment]
Hi,

It's just a trivial bug, forgot to put a limit on unpacked line parts apparently, didn't think to add line to test that case and don't use ttls with these zones myself.
Fixed in a31fdda, should work with any number of extra parameters, including TTL, as intended.

Thanks for reporting.

Note that I wrote this a while ago in python2, which goes EOL later this year, though probably will still be around in a deprecated state for a decade, if not more.
Might update it to python3 at some point (should be rather easy, and netaddr is built-in as ipaddress module there), but not sure if maybe will replace remaining two tinydns setups I have with nsd instead.
